hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoTransferSuite.java

```diff
@@ -249,6 +252,7 @@ private HapiSpec okToRepeatSerialNumbersInWipeList() {
         final var ownerWith4AutoAssoc = "ownerWith4AutoAssoc";
         return defaultHapiSpec("OkToRepeatSerialNumbersInWipeList")
                 .given(
+                        snapshotMode(FUZZY_MATCH_AGAINST_MONO_STREAMS),
```

Comment:
Left here by accident?

Reply:
No, to run in CI. Doesn't it work ?

Reply:
The `itest` and `eet` [`TargetNetworkType`](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/TargetNetworkType.java#L36)'s are `CI_DOCKER_NETWORK`, whose streams are (currently) never fuzzy-matched against snapshots.

But if we want our CI `@HapiTest` to fuzzy-match its streams against the snapshot, we need to use `FUZZY_MATCH_AGAINST_HAPI_TEST_STREAMS`.

Created https://github.com/hashgraph/hedera-services/issues/9811 to eliminate all these extra parameters and properties.

Reply:
Created https://github.com/hashgraph/hedera-services/issues/9812 so we won't have to explicitly opt-in each `HapiSpec` to fuzzy-matching in CI.

Reply:
Got it. Will remove these then 😄  Thank you very much